docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.md

```diff
@@ -5,12 +5,6 @@ import Prerequisites from '../../connecting_to_your_data/components/prerequisite
 
 This guide will help you instantiate a Data Context without a yml file, aka configure a Data Context in code. If you are working in an environment without easy access to a local filesystem (e.g. AWS Spark EMR, Databricks, etc.) you may wish to configure your Data Context in code, within your notebook or workflow tool (e.g. Airflow DAG node).
 
-<Prerequisites>
-
-- [Followed the Getting Started tutorial and have a basic familiarity with the Great Expectations configuration](../../../tutorials/getting_started/intro.md)
-
-</Prerequisites>
-
```

Comment:
I believe that you just need to remove line 10. Removing the `<Prerequisites>` html tags will remove the "Completed the Getting Started Tutorial" and "Have a working installation of Great Expectations" bullets as well.

Reply:
The code says that it's importing Prerequisites from another file. I think the formatting may already be in the import.

Reply:
The import above is importing the `Prerequisites` tag so it can be used within the file. Then the element actually needs to be "called" within the file. That is what contains the first two bullets. It is also possible to add additional bullets by wrapping them in the `Prerequisities` tag. That is the duplication that I believe you are intending to remove here. Removing the `Prerequisites` tag will remove the entire `Prerequisites` element.  

If you build the docs locally, you'll be able to see this in action.

Reply:
I did build the docs locally, but was unable to preview those prereq tags. I thought I saw another document  with the <Prerequisites> section without calling the <Prerequisites> tag, but I can't find that document. Anyway, makes sense. Probably needs those tags.